packages/create-cloudflare/src/cli.ts

```diff
@@ -91,6 +126,7 @@ export const parseArgs = async (argv: string[]): Promise<Partial<C3Args>> => {
 		.option("type", { type: "string" })
 		.option("framework", { type: "string" })
 		.option("deploy", { type: "boolean" })
+		.option("auto-update", { type: "boolean", default: C3_DEFAULTS.autoUpdate })
```

Comment:
can you please add description for this field too, otherwise `--help` output won't be very useful. 

could you also backfill `descriptions` for the other flags that don't have one? thanks